### ISSUE TITLE
Can only see 7 candidates on Receipts / Disbursement screen [dupe of #13188]

### ISSUE THREAD

**Issue by AaronStew** [fecgov/FEC#13189]
[DUPE / EXPANSION OF A BOT-ENTERED FEEDBACK #13188 FROM A SHORT TIME EARLIER]

The bot feedback is from my entry on the FEC website.  I may have gotten myself confused there... will create a new Issue here once I sort that out.  I apologize for any confusion.  Please ignore #13188.

**Comment by PaulClark2** [comment]
ROLLINS, WILL (H2CA42205) and CARAVEO, YADIRA (H2CO08020) are not Senate candidates. Both, Rollins and Caraveo are House candidates. 

Try the all candidates data table: https://www.fec.gov/data/candidates/?q=H2CA42205&q=H2CO08020&q=S2WI00219&q=S4AZ00139&q=S4TX00722&q=S6MT00162&q=S6OH00163&q=S6PA00217&q=S8NV00156&election_year=2024&cycle=2024&election_full=true

**Comment by AaronStew** [comment]
![CAP_ 7241](https://github.com/fecgov/FEC/assets/164087642/fad193c6-accd-4859-9c79-31b392b5ed6c)
Thanks PaulClark2.  I was in parallel beginning to figure that out and vacated my Issue here before seeing your reply.  I'm sure that's it.
